src/Microsoft.DotNet.Wpf/src/Common/Graphics/wgx_core_types.cs

```diff
@@ -454,6 +454,15 @@ internal struct MilColorF
     internal float g;
     internal float b;
     internal float a;
+
+    public override int GetHashCode()
+    {
+        return a.GetHashCode() ^ r.GetHashCode() ^ g.GetHashCode() ^ b.GetHashCode();

```

Comment:
Just to clarify for me, we don't care about uniqueness for these hash codes?

Reply:
If two colors compare as equal, hashes would be equal. If two colors are not  equal, the would be different. 

Are you thinking about uniqueness in a different sense ?

Reply:
If two colors are different the hash codes could be the same, the XOR can cancel out bits. Theres nothing you can do to prevent that, the hash code range has just 32 bit and four floating point values are definitely above 32 bit even if you restrict them to [0,1] ranges or something along those lines.

MilColorF can't have unique hash codes, you were probably thinking about integer based colors, bat that GetHashCode didn't have a bug in the first place because the struct was just 32 bit to begin with and got hashed correctly by coreclr as one unit.

Reply:
Yeah that makes sense, there is no guaranteeing uniqueness here given the dataset.